spec/controllers/hyrax/trophies_controller_spec.rb

```diff
@@ -2,7 +2,13 @@
 RSpec.describe Hyrax::TrophiesController do
   describe "#toggle_trophy" do
     let(:user) { create(:user) }
-    let(:work) { create(:work, user: user) }
+    let(:work) do
+      if Hyrax.config.use_valkyrie?
+        FactoryBot.valkyrie_create(:monograph, depositor: user.user_key)
+      else
+        create(:work, user: user)
+      end
+    end
```

Comment:
In many situations like this using the hyrax_work factory is good enough. Wings will create an AF GenericWork, or when wings is off a resource will be created.
```suggestion
  let(:work) { FactoryBot.valkyrie_create(:hyrax_work, user: user) }
```